### ISSUE TITLE
couldn't get susie converged although LD matrix seems fine

### ISSUE THREAD

**Issue by PhoebeGuo97** [stephenslab/susieR#182]
Hello, I tried fine-mapping some loci of my interest using susie_rss. The LD matrix was calculated using UK biobank as the reference panel. It got converged for some loci easily using default settings, but it never converged for a locus on Chr17q21 . I have tried increasing max_iter to 5000, setting check_prior=FALSE and removing SNPs with logLR>0 but still couldn't fix this issue. 

I have attached the condz$plot here and the lambda is ~0.01. I notice that some SNPs look like outliers in the plot but they are not red dots. Thanks for your help in advance!

![Bell_17RL_susie](https://user-images.githubusercontent.com/99839214/223194360-3de73375-7016-460b-be06-79d49b476252.png)

**Comment by pcarbo** [comment]
Yes, you could try removing these 5 additional SNPs and see if it helps.

**Comment by PhoebeGuo97** [comment]
Hello @pcarbo, I'm wondering how to find these 5 SNPs. I don't think filtering by logLR>0 will do that. Thanks!

**Comment by pcarbo** [comment]
The plot shows `conditional_mean$z` vs. `conditional_dist$condmean`, so you would want to look for rows in the `conditional_dist` data frame that diff a lot in those two columns.

**Comment by PhoebeGuo97** [comment]
I removed 6 SNPs with the largest absolute value of conditional_dist$z_std_diff and susie_rss converged. 5/6 SNPs' logLR are smaller than 0, so I guess maybe that's the reason that they are not shown as red dots in the plot.
